<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newFile</class>
 <widget class="QDialog" name="newFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Photometic Type</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbTypeA">
      <property name="text">
       <string>Type A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbTypeB">
      <property name="text">
       <string>Type B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbTypeC">
      <property name="text">
       <string>Type C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>IESNA:LM-63-2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IESNA:LM-63-1995</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageC">
    <widget class="QGroupBox" name="groupVertical">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Vertical angles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbV0">
        <property name="text">
         <string>The first angle is 0 deg 
 and the last is 90 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbV1">
        <property name="text">
         <string>The first angle is 0 deg 
 and the last is 180 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbV2">
        <property name="text">
         <string>The first angle is 90 deg 
 and the last is 180 deg.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupHorizontal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Horizontal symmetry</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rb0">
        <property name="text">
         <string>Laterally symmetric in all 
photometric planes.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb1">
        <property name="text">
         <string>Symetric in each quadrant.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb2">
        <property name="text">
         <string>Bilaterally symmetric about the 
0-180 deg photometric plane.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb3">
        <property name="text">
         <string>Bilaterally symmetric about the 
90-270 deg photometric plane.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb4">
        <property name="text">
         <string>No symmetry.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_AB">
    <widget class="QGroupBox" name="groupHorizontal_AB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>283</width>
       <height>118</height>
      </rect>
     </property>
     <property name="title">
      <string>Horizontal symmetry</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="rb0_AB">
        <property name="text">
         <string>Laterally symmetric about a 
 vertical reference plane.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb1_AB">
        <property name="text">
         <string>Not laterally symmetric about 
 a vertical reference plane.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupVertical_AB">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>259</width>
       <height>118</height>
      </rect>
     </property>
     <property name="title">
      <string>Vertical angles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbV0_AB">
        <property name="text">
         <string>The first angle is -90 deg 
 and the last is 90 deg.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbV1_AB">
        <property name="text">
         <string>The first angle is 0 deg 
 and the last is 90 deg.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
